ci-build/Dockerfile

```diff
@@ -16,7 +17,10 @@ RUN apt-get update && \
 # - fonts, for when Prince renders to PDF
```

Comment:
Add a note here for the Prince dependencies. A shame we can't get them automatically through the deb anymore...